pkg/epp/scheduling/types/types.go

```diff
@@ -30,23 +30,22 @@ type LLMRequest struct {
 	Model string
```

Comment:
in all places in the code I see that when LLMRequest is used, the fields that are set are Model, ResolvedTargetModel and Critical. I would remove all fields that are not used.

Reply:
```
// LLMRequest is a structured representation of the fields we parse out of the LLMRequest body.
type LLMRequest struct {
	Model string
	// Resolved target model is the final target model after traffic split.
	ResolvedTargetModel string
	Critical            bool
}

func (r *LLMRequest) String() string {
	return fmt.Sprintf("Model: %s, ResolvedTargetModel: %s, Critical: %t", r.Model, r.ResolvedTargetModel, r.Critical)
}
```

Reply:
These fields are all used.